File: Haskell/beginner/1009.hs
-- Doing imports
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    name <- getLine -- Lol, What a weird way to grab a string
    base_salary <- readLn :: IO Double
    sales <- readLn :: IO Double
    
    -- Doing the calculations
    let final_salary = base_salary + (sales * 0.15)

    -- Showing results
    printf "TOTAL = R$ %.2f\n" final_salary

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1009.c
/*Including Libraries*/
#include <stdio.h>

/*Main Function*/
int main(){
    /*Variable declaration and initialization*/
    char name[50];
    double base_salary, final_salary, sales, comission;

    /*Requesting values to user*/
    scanf("%s", name);
    scanf("%lf", &base_salary);
    scanf("%lf", &sales);

    /*Doing the calculations*/
    comission = (sales * 0.15);
    final_salary = base_salary + comission;

    /*Showing results*/
    printf("TOTAL = R$ %.2f\n", final_salary);

    /*Ending algorithm*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/